msobooking.yaml

```diff
@@ -0,0 +1,436 @@
+        type: "publicsafety.employeeid"
+        column: "RELOFF"
+    name: "releaseofficer"
+
+  dnatest:
+    fqn: "ol.medicaltest"
+    entitySetName: "MSODnatest"
+    propertyDefinitions:
+      general.id:
+        type: "general.id"
+        column: "DNA_SAMPLE_DATE"
+      ol.datetimeadministered:
+        type: "ol.datetimeadministered"
+        column: "DNA_SAMPLE_DATE"
+        transforms:
+        - !<transforms.DateAsDateTimeTransform>
+          pattern: ["yyyy-MM-dd"]
+          timezone: "America/New_York"
+      ol.status:
+        type: "ol.status"
+        column: "DNA_SAMPLE_STATUS"
+    name: "dnatest"
+
+  sentence:
+    fqn: "ol.sentence"
+    entitySetName: "MSOSentence"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "SENTENCE"
+      ol.notes:
+        type: "ol.notes"
+        column: "SENTENCE"
+    name: "sentence"
+
+  officer:
+    fqn: "ol.employee"
+    entitySetName: "MSOOfficer"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "IDOFF"
+      publicsafety.employeeid:
+        type: "publicsafety.employeeid"
+        column: "IDOFF"
+    name: "officer"
+
+  dnaofficer:
+    fqn: "ol.employee"
+    entitySetName: "MSOOfficer"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "DNA_SAMPLE_OFFICER"
+      publicsafety.employeeid:
+        type: "publicsafety.employeeid"
+        column: "DNA_SAMPLE_OFFICER"
+    name: "dnaofficer"
+
+  committingofficer:
+    fqn: "ol.employee"
+    entitySetName: "MSOOfficer"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "COMOFF"
+      publicsafety.employeeid:
+        type: "publicsafety.employeeid"
+        column: "COMOFF"
+    name: "committingofficer"
+
+  committingcounty:
+    fqn: "ol.location"
+    entitySetName: "MSOCommittingcounty"
+    propertyDefinitions:
+      general.id:
+        type: "general.id"
+        column: "COUNTY"
+      ol.county:
+        type: "ol.county"
+        column: "COUNTY"
+    name: "committingcounty"
+
+  releaselocation:
+    fqn: "ol.location"
+    entitySetName: "MSOReleaselocation"
+    propertyDefinitions:
+      general.id:
+        type: "general.id"
+        column: "RELLOC"
+      location.name:
+        type: "location.name"
+        column: "RELLOC"
+      general.notes:
+        type: "general.notes"
+        column: "RELADDR1"
+    name: "releaselocation"
+
+  immigrationstatus:
+    fqn: "ol.immigrationstatus"
+    entitySetName: "MSOImmigrationstatus"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "INS"
+      ol.status:
+        type: "ol.status"
+        column: "INS"
+    name: "immigrationstatus"
+
+  timeserved:
+    fqn: "ol.jailstaylength"
+    entitySetName: "MSOTimeserved"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "SERVED"
+      criminaljustice.timeserveddays:
+        type: "criminaljustice.timeserveddays"
```

Comment:
It looks like int properties don't have parsers in this flight. Do they still need them? This one, and age below. Maybe others.

Reply:
Int parsers are no longer necessary. Nullable ints can be stored in pandas by casting the column .astype('Int64'). This will write correctly to a cleaned table with .to_sql and then integrate without need for the transform.